### ISSUE TITLE
Dynamic color theme?

### ISSUE THREAD

**Issue by Eyt-Lev** [Sternbach-Software/BeautifulZmanim#1]
replacing this code in Theme.kt
```
val colors = if (!useDarkTheme) {
    LightColorScheme
} else {
    DarkColorScheme
}
```
with that
```
val dynamicColor = Build.VERSION.SDK_INT >= Build.VERSION_CODES.S
val colors = when {
    dynamicColor && useDarkTheme -> dynamicDarkColorScheme(LocalContext.current)
    dynamicColor && !useDarkTheme -> dynamicLightColorScheme(LocalContext.current)
    useDarkTheme -> DarkColors
    else -> LightColors
}
```
allows the app to use [Material 3 Dynamic color](https://m3.material.io/styles/color/dynamic-color/overview) and i think that can help a lot.

**Comment by Sternbach-Software** [comment]
Thank you! I will have to think about it. I'm not personally a huge fan of material 3 over 2, but it _is_ the modern design language. I think it also requires extra dependencies across other parts of the app, and I am not sure if it is KMP-friendly yet.

**Comment by Sternbach-Software** [comment]
Implemented in 24ab84be
